app/models/publication_log.rb

```diff
@@ -12,7 +12,7 @@ class PublicationLog
 
   alias_attribute :published_at, :created_at
 
-  scope :with_slug_prefix, ->(slug) { where(slug: /^#{slug}.*/) }
+  scope :with_slug_prefix, ->(slug) { where(slug: /^#{slug}(\/|$)/) }
```

Comment:
We don't seem to have change notes on manual editions at the moment so I'm not sure if we strictly need the alternative match on the end of the slug string (`$`). But it makes sense to have it as these could easily be added later.